ibis/tests/all/test_temporal.py

```diff
@@ -36,14 +36,24 @@ def test_date_extract(backend, alltypes, df, attr):
 
 
 @pytest.mark.parametrize(
-    'attr', ['year', 'month', 'day', 'hour', 'minute', 'second']
+    'attr', ['year', 'month', 'day', 'epoch', 'hour', 'minute', 'second']
 )
 @pytest.mark.xfail_unsupported
 def test_timestamp_extract(backend, alltypes, df, attr):
     expr = getattr(alltypes.timestamp_col, attr)()
-    expected = getattr(df.timestamp_col.dt, attr).astype('int32')
+
+    if attr == 'epoch':
+        expected = pd.Series(
+            (pd.DatetimeIndex(df.timestamp_col) - pd.Timestamp('1970-01-01'))
+            // pd.Timedelta('1s')
+        )
+    else:
+        expected = getattr(df.timestamp_col.dt, attr.replace('_', '')).astype(
```

Comment:
Is the replace needed? Doesn't feel like it is.

Reply:
it is not needed here .. but as I am doing that in other PRs  .. and these probably will be merged before this one .. I just kept the same code.